website/docs/cloud-docs/workspaces/settings/vcs.mdx

```diff
@@ -65,6 +65,16 @@ You can use trigger patterns and trigger prefixes in the **Add path** field to s
 
 -> **Note:** Terraform Cloud triggers runs on all attached workspaces if it does not receive a list of changed files or if that list is too large to process. When this happens, Terraform Cloud may show several runs with completed plans that do not result in infrastructure changes.
 
+#### Trigger runs when a git tag is published
+
+This option instructs Terraform to begin new runs only for changes that have a specific tag format.
+
+The tag format can be chosen between the following options:
+- **Semantic Versioning:** It matches tags in the popular [SemVer format](https://semver.org/). (e.g. 0.4.2)
```

Comment:
Do we want to give the regex for these three values?

Reply:
Done